homeassistant/components/ibeacon/coordinator.py

```diff
@@ -141,6 +145,14 @@ def __init__(
         # iBeacons with random MAC addresses, fixed UUID, random major/minor
         self._major_minor_by_uuid: dict[str, set[tuple[int, int]]] = {}
 
+        # iBeacons from devices with no name
+        self._allow_nameless_uuids = set(
+            entry.options.get(CONF_ALLOW_NAMELESS_UUIDS, [])
+        )
+        self._ignored_nameless_by_uuid: dict[str, set[str]] = {}
+
+        self._entry.add_update_listener(self.async_config_entry_updated)
```

Comment:
This leaks an update listener because its never unsubscribed

Reply:
Fixed.